### ISSUE TITLE
searching in contacts and favorite contacts do not work

### ISSUE THREAD

**Issue by greskom** [ubports/ubuntu-touch#1699]
- Device: Volla, BQ Aquaris E4.5
- Channel: stable
- Build: OTA-16

### Steps to reproduce

Try to search for contacts or switch to favorite contacts.

### Expected behavior

Contacts are filtered or favorite contacts are displayed.

### Actual behavior

Only full list of contacts is returned.

**Comment by lduboeuf** [comment]
can't reproduce on devel (BQ4.5)

**Comment by cibersheep** [comment]
This issue might be moved into:
Dielaer https://github.com/ubports/dialer-app/issues check [156](https://github.com/ubports/dialer-app/issues/156)
or
Contacts https://github.com/ubports/address-book-app/issues Check [175](https://github.com/ubports/address-book-app/issues/175)

**Comment by lduboeuf** [comment]
yes, the contact backend is slower now, idk why. Something interesting would be to test if search contact with dial pad has the same issue, but not already merged yet: https://github.com/ubports/dialer-app/pull/158

**Comment by Danfro** [comment]
I can confirm [Xperia X, devel], that tapping on favorites does often not open the list of favorites, but just gives the "normal" contact list. At one occasion I checked the log: [pastebin](http://paste.ubuntu.com/p/CX4z4XqNqz/). 

A restart of the app seems to solve this. Then tapping on contacts does work as expected.

 In my case this seems to have worsened after Qt5.12. But I admit this is more a feeling than proper statistics. ;-)

**Comment by Thieb67k** [comment]
I can confirm what it said here.
I would add that in my case, searching for a contact will gives (not always) the full list of contacts (a-->z), but, some of them are hilight in bold type (according to the search field).

Happens on mako, OTA-19

**Comment by lduboeuf** [comment]
can someone test on the new devel if its still there ?
